packages/zoe/test/unitTests/contracts/gimix/test-postalSvc.js

```diff
@@ -0,0 +1,144 @@
+// @ts-check
+// eslint-disable-next-line import/order
+import { test as anyTest } from '../../../../tools/prepare-test-env-ava.js';
+
+import { createRequire } from 'module';
+
+import { E, Far } from '@endo/far';
+import { unsafeMakeBundleCache } from '@agoric/swingset-vat/tools/bundleTool.js';
+import { makeZoeKitForTest } from '../../../../tools/setup-zoe.js';
+import { makeNameHubKit, makePromiseSpace } from '@agoric/vats';
+import { makeWellKnownSpaces } from '@agoric/vats/src/core/utils.js';
+import { AmountMath } from '@agoric/ertp/src/amountMath.js';
+import { startPostalSvc } from '../../../../src/contracts/gimix/start-postalSvc.js';
+
+const { Fail } = assert;
+
+/** @type {import('ava').TestFn<Awaited<ReturnType<makeTestContext>>>} */
+const test = anyTest;
+
+const myRequire = createRequire(import.meta.url);
+/** @param {string} specifier */
+const asset = specifier => myRequire.resolve(specifier);
+
+/**
+ * Wrap Zoe exo in a Far so that we can override methods.
+ *
+ * @param {ZoeService} zoeMethods
+ */
+const wrapZoe = zoeMethods => {
```

Comment:
to override `installBundleID`

p.s. thanks for taking a look.

Reply:
I don't get it. It looks like it passes `installBundleID()` through just like it does for all the other methods it supports. In a test, why is this better than just passing `zoeMethods` through?  Did I misread something?

Reply:
the real zoe is an exo, so you can't just go `{ ...zoe, installBundleID: myMockInstallBundle }`. (I should probably document what happens if you try.)

`wrapZoe` takes the zoe exo and gives you back a plain record/object, where you _can_ override methods using spread expressions like that:

```js
    const zoe = Far('ZoeService', {
      ...wrapZoe(zoeService),
      installBundleID: install1BundleID,
    });
```

Reply:
Just so you're aware of `bindAllMethods` https://github.com/Agoric/agoric-sdk/blob/6d9ae67c57499d7139dfce019b8c76f9ecdb825f/packages/internal/src/method-tools.js#L77

As it says there, `bindAllMethods` was only a transitional measure until replaced by code this PR's `wrapZoe`, so no change suggested. Just FYI.

Reply:
Thanks; I was only vaguely aware of it. But I'm leery to use stuff from `@agoric/internal`, because it's a private package, and I was thinking this contract should probably be outside the SDK.